Formula/loudmouth.rb

```diff
@@ -24,11 +23,25 @@ class Loudmouth < Formula
   depends_on "glib"
   depends_on "libidn"
   depends_on "gnutls"
+  depends_on "gettext"
 
   def install
     system "./autogen.sh", "-n" if build.head?
     system "./configure", "--disable-debug", "--disable-dependency-tracking",
                           "--prefix=#{prefix}", "--with-ssl=gnutls"
+    system "make"
+    system "make", "check"
     system "make", "install"
+    (pkgshare/"examples").install Dir["examples/*.c"]
+  end
+
+  test do
+    cp pkgshare/"examples/lm-send-async.c", testpath
+    system ENV.cc, "lm-send-async.c", "-o", "test",
+      "-lloudmouth-1", "-lglib-2.0",
+      "-I#{include}/loudmouth-1.0",
+      "-I#{Formula["glib"].include}/glib-2.0",
```

Comment:
Should possibly be `.opt_include` so we're consistent with a lot of other usage.